tools/xmc-flasher.py

```diff
@@ -112,9 +115,16 @@ def process_console_output(string):
         if string in line and '=' in line:
             print(line)
 
+def check_serial_number(serial_num):
+    if serial_num == None:
+        raise Exception("Device not found! Please check the COM port")
```

Comment:
"COM" is very Windows-specific. Can we write "serial port"?

Reply:
Fixed